submarine-commons/commons-metastore/pom.xml

```diff
@@ -318,19 +326,6 @@
                 </filter>
               </filters>
               <relocations>
-                <relocation>
```

Comment:
Why do we need to delete this relocation configuration?
It is used to build a shade jar